scripts/zones/Leujaoam_Sanctum/IDs.lua

```diff
@@ -27,11 +27,21 @@ zones[xi.zone.LEUJAOAM_SANCTUM] =
         ASSAULT_START_OFFSET          = 7454, -- Max MP Down removed for <name>.
         TIME_TO_COMPLETE              = 7515, -- You have <number> [minute/minutes] (Earth time) to complete this mission.
         MISSION_FAILED                = 7516, -- The mission has failed. Leaving area.
-        RUNE_UNLOCKED_POS             = 7517, -- ission objective completed. Unlocking Rune of Release ([A/B/C/D/E/F/G/H/I/J/K/L/M/N/O/P/Q/R/S/T/U/V/W/X/Y/Z]-#).
+        RUNE_UNLOCKED_POS             = 7517, -- Mission objective completed. Unlocking Rune of Release ([A/B/C/D/E/F/G/H/I/J/K/L/M/N/O/P/Q/R/S/T/U/V/W/X/Y/Z]-#).
```

Comment:
These aren't typos, these are artifacts of the UpdateExtractor client update process. If you change these, automated client updates will fail

https://github.com/LandSandBoat/UpdateExtractor/blob/5f5e90250d63c71292bcdef30cd5c34f2df05c8e/out/scripts/zones/Leujaoam_Sanctum/Text.lua.raw.txt#L7434-L7435